webhook_test.go

```diff
@@ -193,6 +194,28 @@ func TestClustersMainline(t *testing.T) {
 	Expect(cdsResp.Clusters[1].Name).To(Equal(AuthZClusterName))
 }
 
+func TestClustersMainlineNoChange(t *testing.T) {
+	RegisterTestingT(t)
+
+	configOptions.SetCluster = false
```

Comment:
No need to fix (see top level review comment), but I don't think setting global variables in individual test cases will be safe when tests are run in parallel on multiple cores.  We'd want to make the webservice a struct and attach the options to it.

Reply:
yes. agreed.